benchexec/tools/jpf-core.py

```diff
@@ -0,0 +1,60 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2018  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for JPF (plain jpf-core)
+    (https://github.com/javapathfinder/jpf-core/).
+    """
+
+    REQUIRED_PATHS = [
+                  "jpf-core/bin",
+                  "jpf-core/build",
+                  "jpf-sv-comp"
+                  ]
+    def executable(self):
+        return util.find_executable('jpf-core-sv-comp')
+
+
+    def version(self, executable):
+        return 'rev '+self._version_from_tool(executable).split()[2]
+
+
+    def name(self):
+        return 'JPF'
```

Comment:
Is there a JPF (other than SPF) that is not JPF-core? If so, then I'd return `JPF-core` here.